src/riscv/lib/src/pvm/common.rs

```diff
@@ -261,8 +261,10 @@ impl<MC: MemoryConfig, BCC: BlockCacheConfig, B: block::Block<MC, M>, M: state_b
         // We want to just slightly modify the state without interfering with normal execution.
         // Increasing the number of instructions executed is a good candidate.
```

Comment:
This comment is now out of date.

Reply:
I removed the outdated line [here](https://github.com/tezos/riscv-pvm/compare/9ffdf80c46928fc0403a793ff14a611d4923ac49..872acbb30f8a00d6ceec85df641d883c47787ee4).